Basic tests

fjsj opened the pull request:
Based on configs from https://github.com/carltongibson/django-filter

fjsj commented on admin_totals/admin.py:
```diff
@@ -1,5 +1,5 @@
 # coding=utf-8
-from __future__ import print_function, unicode_literals, division
+from __future__ import division, print_function, unicode_literals
```
isort made this

fjsj commented on admin_totals/admin.py:
```diff
@@ -13,7 +13,8 @@ def get_results(self, *args, **kwargs):
                 if field in list_totals:
-                    self.aggregations.append(self.result_list.aggregate(agg=list_totals[field](field))['agg'])
+                    self.aggregations.append(
+                        self.result_list.aggregate(agg=list_totals[field](field))['agg'])
```
for flake8 to be happy

fjsj commented on admin_totals/templatetags/admin_totals.py:
```diff
@@ -1,7 +1,9 @@
-from django.contrib.admin.templatetags.admin_list import result_headers, result_hidden_fields, results
+from django.contrib.admin.templatetags.admin_list import (result_headers,
+                                                          result_hidden_fields,
+                                                          results)
```
isort made this

douwevandermeij commented:
Wow, great! Thanks for your contribution!
I'll read through the changed files and merge it later on.

douwevandermeij commented:
Could you also update the readme a bit to explain how to run tests?

fjsj commented:
@douwevandermeij done. I've added the Travis and Codecov badges to README too, but you need to sign up there and authorize the repo, since I don't have write permission for this repo. LMK if you need any help. 

Also, added a test for the template.

douwevandermeij commented:
Thanks, great work!
